.taplo.toml

```diff
@@ -0,0 +1,11 @@
+# See https://taplo.tamasfe.dev/configuration/file.html
+# and https://taplo.tamasfe.dev/configuration/formatter-options.html
+
+[formatting]
+# This is a matter of taste, but expanded inline arrays make tables harder to read.
+array_auto_collapse = true
+inline_table_expand = false
```

Comment:
Not sure about setting `inline_table_expand` to `false`. The expansion **only** happens when the line exceeds the length defined in `column_width` (which we should have at 100).

With it `false`, the following happens:

```toml
winapi = { version = "0.3.9", features = [
    "winbase",
    "libloaderapi",
    "errhandlingapi",
    "winuser",
    "shellscalingapi",
    "shobjidl",
    "combaseapi",
    "dxgi1_3",
    "dwmapi",
    "wincon",
    "fileapi",
    "processenv",
    "winbase",
    "winerror",
    "handleapi",
    "shellapi",
    "winnls",
] }

# .. gets converted into:

winapi = { version = "0.3.9", features = ["winbase", "libloaderapi", "errhandlingapi", "winuser", "shellscalingapi", "shobjidl", "combaseapi", "dxgi1_3", "dwmapi", "wincon", "fileapi", "processenv", "winbase", "winerror", "handleapi", "shellapi", "winnls"] }
```

Also, just so I understand your claim, are you saying that the following expansion that we currently have makes the `[workspace.lints]` table hard to read?

```toml
rust.unexpected_cfgs = { level = "warn", check-cfg = [
    'cfg(FALSE)',
    'cfg(tarpaulin_include)',
] }
```

Reply:
I've found while testing it that the formatter was really eager to break up inline tables even when it didn't seem needed to stay below the column limit. I'll give it another look.